src/lib/setupContext.js

```diff
@@ -183,8 +182,7 @@ function trackModified(files) {
   let changed = false
 
   for (let file of files) {
-    let pathname = url.parse(file).pathname
-    let newModified = fs.statSync(pathname).mtimeMs
+    let newModified = fs.statSync(file).mtimeMs
```

Comment:
Another way we can fix this is by decoding the URI component.

```js
    let pathname = url.parse(file).pathname
    let newModified = fs.statSync(decodeURIComponent(pathname)).mtimeMs
```

but I think just that `file` variable will also work, I'm not sure about why url.parse is used here.

Reply:
Cool! Okay i will then change it to use `decodeURIComponent`